src/spec/integration/export.spec.coffee

```diff
@@ -135,11 +137,64 @@ describe 'Export integration tests', ->
+    expectedProduct = 'false,false,ImpEx with all types,1,MONGO_ID,,,,,,Foo,,foo,,,,'
+    expectedVariant =   ',,,2,,123,,,,,,,,,,,,,,,lenum1,,,,,,,,,,,,,,,,,,,lenum1;lenum2'
+
+    @export.exportFull(file)
+    .then (result) ->
+      expect(result).toBe 'Export done.'
+
+      try
+        fs.statSync(file).isFile()
+        return Promise.resolve()
+      catch err
+        return Promise.reject "Archive was not generated"
+    .then ->
+      console.log "Archive was generated successfully"
+      extractArchive(file, {dir: '/tmp'})
+    .then ->
+
+      try
+        fs.statSync '/tmp/products'
+        exportedFolder = '/tmp/products/'
+        files = fs.readdirSync(exportedFolder)
+        expect(files.length).toBe 1
+        csvFile = exportedFolder + files[0]
+        fs.statSync csvFile
```

Comment:
statSync will tell you if that file exists (was created) .. if not it will throw an error